### PR TITLE
Add layouts-panel-list with tests

### PR DESCRIPTION
**Pull request by nicucampian**
-add layouts-panel-component
-add unit test
-add integration test
-add layout images assets
-add scss for component
-sendAction on layoutClick event with shapes (array) parameter.

### CODE REVIEW

File: app/components/layouts-panel-list/component.js [hunk 1]
```diff
@@ -0,0 +1,288 @@
...
+export default Ember.Component.extend({
+  tagName: 'layouts-panel',
+  classNames: ['layouts-panel'],
+  layouts: [],
+  init() {
+    this._super(...arguments);
+    this.layouts = bringLayouts;
+  },
+  actions: {
+    layoutClick(shapes) {
```
Comment (andreisebastianc) [inline app/components/layouts-panel-list/component.js]: get rid of this action

**Comment by andreisebastianc** [comment]
overall looks good; other opinions ?

**Comment by vladMararu** [comment]
I agree!

File: app/components/layouts-panel-list/template.hbs [hunk 2]
```diff
@@ -0,0 +1,6 @@
+{{yield}}
+{{#each layouts as |layout|}}
+  <button {{action "layoutClick" layout.shapes}}>
```
Comment (andreisebastianc) [inline app/components/layouts-panel-list/template.hbs]: extract this to a generic button with icon component

**Pull request merged**